lib/audited/auditor.rb

```diff
@@ -196,10 +196,22 @@ def revision_with(attributes)
 
       def audited_changes
         all_changes = respond_to?(:changes_to_save) ? changes_to_save : changes
-        if audited_options[:only].present?
-          all_changes.slice(*self.class.audited_columns)
-        else
-          all_changes.except(*self.class.non_audited_columns)
+        filtered_changes = \
+          if audited_options[:only].present?
+            all_changes.slice(*self.class.audited_columns)
+          else
+            all_changes.except(*self.class.non_audited_columns)
+          end
+
+        filtered_changes = normalize_enum_changes(filtered_changes)
+        filtered_changes.to_hash
```

Comment:
why the `to_hash`? shouldn't this already be a hash?

Reply:
This is a hash-like object (`ActiveSupport::HashWithIndifferentAccess`)

Reply:
@fatkodima a little late to the party, but this object indeed was an `ActiveSupport::HashWithIndifferentAccess`, but after you call `to_hash` on it, it's not anymore, it's simply a `Hash` object.  The reason I'm writing this is that after upgrading from 4.8.0 to 4.9.0 I noticed that i cannot access some audited data by simbol, because they are not stored as `ActiveSupport::HashWithIndifferentAccess` anymore. Was this intended?